src/docfx.website.themes/docs.html/partials/class.tmpl.partial

```diff
@@ -286,7 +286,7 @@
 <div class="level">
   <section class="primary">
      {{#type.specName}}
-        {{{value}}}
+        <code class="lang-{{lang}}">{{{value}}}</code>
```

Comment:
how about update together?

---
In reply to: [67851987](https://github.com/dotnet/docfx/pull/409#discussion_r67851987) [](ancestors = 67851987)